### DISCUSSION TITLE
Metrics for experimental retry

### DISCUSSION THREAD

**Discussion by nikola-mladenovic** [apollographql/router#2734] [Q&A] [answered]
Hi all,

Regarding the experimental retry https://www.apollographql.com/docs/router/configuration/traffic-shaping/ - is there a way to get logging / metrics on when the retries occur, in order to gain more insight, and adjust params if needed?

**Comment by bnjjj** [comment]
I created an [issue](https://github.com/apollographql/router/issues/2736)

**Reply by bnjjj** [reply]
(there is no way to know it right now)

**Comment by abernix** [comment, marked as answer]
It is now possible as of https://github.com/apollographql/router/releases/tag/v1.13.1.
